drivers/flash/flash_realtek_rts5912.c

```diff
@@ -751,6 +775,12 @@ static int flash_rts5912_ex_op(const struct device *dev, uint16_t opcode, const
 	case FLASH_RTS5912_EX_OP_RD_SR2:
 		ret = flash_read_sr2(dev, (uint8_t *)in);
 		break;
+	case FLASH_RTS5912_EX_OP_SET_WP:
+		ret = flash_set_wp(dev, (uint8_t *)out);
+		break;
+	case FLASH_RTS5912_EX_OP_GET_WP:
+		ret = flash_get_wp(dev, (uint8_t *)in);
+		break;
```

Comment:
in and out are backwards, I think (set should use "in", get should put data in "out")